src/commands/create_command.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, schema_file):
     def execute(self):
         os.makedirs(self.path, exist_ok=True)
         for table in self.data[Keys.TABLES]:
-            t_path = os.path.join(self.path, table[Keys.NAME])
-            os.makedirs(t_path, exist_ok=True)
+            table = TableMetaData(table)
```

Comment:
This is confusing between tableMetadata and table in the loop. I recommend rename this variable as table_metadata = TableMetadata(table)